boards/x86/intel_ish/Kconfig.defconfig

```diff
@@ -0,0 +1,33 @@
+# Copyright (c) 2023 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+if BOARD_INTEL_ISH_5_4_1 || BOARD_INTEL_ISH_5_6_0 || BOARD_INTEL_ISH_5_8_0
+
+config BOARD
+	default "intel_ish_5_4_1" if BOARD_INTEL_ISH_5_4_1
+	default "intel_ish_5_6_0" if BOARD_INTEL_ISH_5_6_0
+	default "intel_ish_5_8_0" if BOARD_INTEL_ISH_5_8_0
+
+if TEST
+config TEST_EXTRA_STACK_SIZE
+	int
+	default 1024
+endif # TEST
+
+config HPET_TIMER
+	default y
+
+config SYS_CLOCK_TICKS_PER_SEC
+	default 2048 if HPET_TIMER # HPET is 32768 HZ
+
+if I2C
+
+# I2C logging is used for ISH. Better to get I2C Device ready early
+config I2C_INIT_PRIORITY
```

Comment:
Can you explain the comment a bit more here about i2c logging? seems the console and log are setup with a uart

Reply:
just remove currently. to add back when enabling ISH I2C logging.